Earthfile

```diff
@@ -818,3 +818,25 @@ npm-update-all:
         RUN cd $nodepath && npm update
         SAVE ARTIFACT --if-exists $nodepath/package-lock.json AS LOCAL $nodepath/package-lock.json
     END
+
+# merge-main-to-docs merges the main branch into docs-0.7
+merge-main-to-docs:
+    FROM alpine/git
+    # Ok, if you have no choice
+    ARG git_url="git@github.com:earthly/earthly"
+    GIT CLONE "$git_url" earthly
+    RUN --mount=type=secret,id=littleredcorvette-id_rsa,mode=0400,target=/root/.ssh/id_rsa \
+          ssh-keyscan github.com >> ~/.ssh/known_hosts && \
+        rm -rf earthly &&\
```

Comment:
what's the reason for `rm -rf earthly`, can we just remove this entirely as well as remove the `GIT CLONE ...` command above?

Reply:
I actually meant to ask you about this, I was following our guidelines, wasn't sure if and how this is beneficial:
https://docs.earthly.dev/best-practices#git-clone-vs-run-git-clone